sources/PTYTextView.h

```diff
@@ -419,5 +427,11 @@ typedef enum {
                          prefix:(NSString *)prefix
                          suffix:(NSString *)suffix;
 
+// Get the transparency value for foreground or background windows.
```

Comment:
I think the comment means it gets the transparency value for this view, which may vary depending on whether the window is active or not. Can you clarify?